src/main/scala/cromwell/engine/db/package.scala

```diff
@@ -14,6 +14,6 @@ package object db {
   // Uniquely identify an entry in the execution table
   case class ExecutionDatabaseKey(fqn: FullyQualifiedName, index: ExecutionIndex)
   // jesId and jesStatus should have stronger types
-  type JesId = Int
-  type JesStatus = String
+  type JesId = Option[String]
```

Comment:
If you're in here monkeying around anyways this might be a good time to make good on that comment on line 16 :)